.github/workflows/binaries.yml

```
@@ -0,0 +1,34 @@
+name: Binaries
+
+on:
+  workflow_dispatch:
+
+jobs:
+  binary:
+    name: Create
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - os: macos-13
+            name: mac-intel
+          - os: macos-14
+            name: mac-arm
+          - os: ubuntu-latest
```

```suggestion
          - os: ubuntu-20.04
```
to avoid surprises if `latest` changes and suddenly our binaries stop working for people overnight.